src/main/java/edu/cornell/kfs/sys/util/CuJsonUtils.java

```diff
@@ -0,0 +1,60 @@
+package edu.cornell.kfs.sys.util;
+
+import java.util.Map;
+import java.util.TimeZone;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public final class CuJsonUtils {
+
+    public static ObjectMapper buildObjectMapperUsingDefaultTimeZone() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setTimeZone(TimeZone.getDefault());
+        return objectMapper;
+    }
+
+    @SafeVarargs
+    public static String buildJsonStringFromEntries(Map.Entry<String, String>... entries) {
+        try {
+            ObjectMapper objectMapper = buildObjectMapperUsingDefaultTimeZone();
+            ObjectNode jsonRoot = objectMapper.createObjectNode();
+            for (Map.Entry<String, String> entry : entries) {
+                jsonRoot.put(entry.getKey(), entry.getValue());
+            }
+            return objectMapper.writeValueAsString(jsonRoot);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static String rebuildJsonStringWithPropertyOverride(
```

Comment:
Is this function needed?  It looks like it is only used in testing.  I am not sure if we would need the functionality. 

The current functionality adds new elements to existing json.  The name implies to me that it will change the value of an existing element in the json.  If you think this function is needed for this PR, I think it should be renamed.

Reply:
This method and the buildJsonStringFromEntries() method are meant for unit testing convenience instead of Production code, though the buildObjectMapperUsingDefaultTimeZone() method is used in Production code. I will create a separate CuJsonTestUtils class to distinguish between the two categories.

As for this method's functionality, it will overwrite the JSON property if it exists, otherwise it will add it to the object. I'll see if I can rename this to make the intent clearer.